pnpm-workspace.yaml

```diff
@@ -5,6 +5,9 @@ packages:
   - packages/*
   - private/*
 
+configDependencies:
```

Comment:
`pnpm-plugin-sku` doesn't really affect the sku repo at all, and it means internal testing/usage of `sku` won't see the warnings added in this PR.